### ISSUE TITLE
Error undefined not allowed for Lightness in oklch()

### ISSUE THREAD

**Issue by equinusocio** [fpetrakov/stylelint-gamut#18]
Hello, i'm using this stylelint plugin with PostCSS but I'm getting an error when I have other postcss plugin to generate colors. See this example:

```postcss
color: oklch(token(--color-primary-40) / 5%)
```

In my case `token()` is replaced by a PostCSS plugin at buildtime, but `stylelint-gamut` is complaining about the 'wrong' syntax giving the error `Error undefined not allowed for Lightness in oklch()`

Any idea how to avoid this?

**Comment by fpetrakov** [comment]
Hello, to avoid it we should update `check` function in src/index.js and ignore oklch declarations that contain `token` string or something like that

**Comment by equinusocio** [comment]
Since there are many postcss plugins and ways to define colors, is probably future-proof to disable the check for color functions that don't start with a number, this would include also `oklch(var())`...

**Comment by fpetrakov** [comment]
Sounds like a solid solution :) You are free to open PR

**Comment by equinusocio** [comment]
Any starting point to suggest? I'm not a javascript developer, I don't know the Stylelint environment and there is not any CONTRIBUTING file. 

It seems the only line to be changed is this
https://github.com/fpetrakov/stylelint-gamut/blob/master/src/index.js#L41

**Comment by equinusocio** [comment]
Just tried:
https://github.com/fpetrakov/stylelint-gamut/pull/19

**Comment by equinusocio** [comment]
Any chance to get this merged?

**Comment by fpetrakov** [comment]
@equinusocio thanks for your work, released it yesterday
![image](https://github.com/fpetrakov/stylelint-gamut/assets/34133492/2647017f-aa74-459f-8694-05a7f9d11d41)
